src/api/forms/service.js

```diff
@@ -237,6 +239,7 @@ export async function createDraftFromLive(formId, author) {
 
   // Copy the draft form definition
   await draftFormDefinition.createDraftFromLive(formId)
+  logger.info(`Draft form definition created for form ID ${formId}`)
```

Comment:
Should we log this message from within `.createDraftFromLive()` instead?

Reply:
Done

Reply:
Can you double check?

You've moved the logging for `.create()` into the function via https://github.com/DEFRA/forms-manager/pull/148#discussion_r1617489552

But not for `.createDraftFromLive()` or `.createLiveFromDraft()`